### PR TITLE
support for Black Duck 2020.2.0 in operator ui

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1241#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/actions/blackducks.go [hunk 1]
```diff
@@ -218,7 +218,7 @@ func (v BlackducksResource) common(c buffalo.Context, bd *blackduckapi.Blackduck
 
 	// environment variables
 	if bd.View.Environs == nil {
-		env := containers.GetHubKnobs()
+		env := containers.GetHubKnobs(bd.Spec.Version)
```
Comment (mphammer) [inline cmd/operator-ui/actions/blackducks.go]: Why is this now needed?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1241#pullrequestreview-362067873" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1241#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/actions/blackducks.go (hunk 1 above)
Comment (msenmurugan) [reply cmd/operator-ui/actions/blackducks.go]: we changed GetHubKnobs method to get the version as input to support SAML verification

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1241#issuecomment-589196473" title="LGTM">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1241#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [mphammer,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**